.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    "extends": "airbnb-base",
+    "plugins": [
+        "import"
+    ]
+};
```

Comment:
Maybe it makes sense to just use the standards set here:
http://cgit.drupalcode.org/drupal/tree/core/.eslintrc?h=8.1.x

This will allow us some consistency in www and zoolander, as they'd both be based on drupal standards. 

I still don't see a way to skip errors though...